dts/arm/st/stm32f091.dtsi

```diff
@@ -5,3 +5,19 @@
  */
 
 #include <st/stm32f0.dtsi>
+
+/ {
+	soc {
+		pinctrl: pin-controller {
+
+			gpioe: gpio@48001000 {
```

Comment:
Is there any difference to the gpioe already defined in stm32f0.dtsi?

Reply:
good catch, thanks. This one does not belong here.

Reply:
To be exact:
This one is correct, wrong one was gpioe in stm32f0.dtsi and it is now corrected